### PR TITLE
Feature/ Clean up Gemfile, remove redundant gems

### PR DESCRIPTION
**Pull request by bagiratania98**
Task https://trello.com/c/12xI6Y5x/20-clean-up-gemfile-remove-redundant-gems

1. Remove redundant gems
2. Update Gemfile

### CODE REVIEW

**Review by stasfly — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -1,66 +1,44 @@
...
   gem 'listen', '~> 3.2'
   gem 'web-console', '>= 3.3.0'
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
-  # linters
+
   gem 'brakeman', '~> 5.2.3', require: false
   gem 'overcommit', '~> 0.59.1'
```
Comment (MishaKlymenko) [inline Gemfile:34]: what is the reason for using a gem with a major version less than 1?

File: Gemfile [hunk 2]
```diff
@@ -1,66 +1,44 @@
...
-# Use Active Storage variant
-# gem 'image_processing', '~> 1.2'
-
-# Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
 
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'pry', '~> 0.14.1'
```
[the remark this answers is not in the document]
Comment (bagiratania98) [reply Gemfile:22]: are there versions greater than 0? I can't find a higher one.

File: Gemfile (hunk 1 above)
Comment (bagiratania98) [reply Gemfile:34]: I didn't write it, but I can fix it if it needs to be changed

**Review by mmrshk — APPROVED** [review]
lgtm

File: Gemfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (mmrshk) [reply Gemfile:22]: pry could be without version, it's a dev tool, not a tool for logic

**Pull request merged**